src/nti/site/folder.py

```diff
@@ -56,3 +61,9 @@ def __repr__(self):
             return super(HostPolicySiteManager, self).__repr__()
         except ConnectionStateError:
             return object.__repr__(self)
+
+
+@component.adapter(IHostPolicyFolder)
+@interface.implementer(IPrincipalRoleManager)
+class SitePrincipalRoleManager(AnnotationPrincipalRoleManager):
```

Comment:
I don't think this belongs here. This package doesn't have a dependency on zope.securitypolicy.

Reply:
Yep, sorry, that was an inadvertant commit.